labellab-flask/api/middleware/logs_decorator.py

```diff
@@ -169,9 +170,23 @@ def wrap(*args, **kwargs):
         project_id = issue['project_id']
         project = find_by_project_id(project_id)
         message = f'{user["username"]} posted a new comment on {project["project_name"]}'
-        category = 'comments'
-        
+        category = 'issues'
 
+      if url.startswith('/api/v1/comment/comment_info/'):
```

Comment:
I did say in the last meeting that we don't need logs for these 3 endpoints. You can remove them